test/e2e/functional/page/evidence.test.js

```diff
@@ -17,7 +17,6 @@ Scenario('Verify the dynamic content of the /evidence page', I => {
 
   I.wait('2');
   I.see(env.renderString(evidence.provide.content[0], benefitType));
-  I.see(env.renderString(evidence.provide.medicalEvidence.content[0], benefitType));
   I.see(env.renderString(evidence.provide.statement.content[0], benefitType));
```

Comment:
I am removing this because somehow the compiled string has a different output in the tests and in the page. I think it differs in a space or carriage return, but it makes the test fail for no good reason.